### ISSUE TITLE
Required input format and processed level

### ISSUE THREAD

**Issue by chetan95** [raktim-ghosh/poldecomp#3]
What type of input format the ".tif" should be and it should be calibrated or calibrated and terrain corrected also, what corrections should be applied before?

**Comment by raktim-ghosh** [comment]
Hi Chetan, the terrain correction is generally applied after incorporating polarimetric decomposition. The 'tif' is generally used for geospatial data. You can extract the array from this format as well with gdal python library. Yeah, if you incorporate polarimetric calibration before, it is better. But for the Full Polarimetric setting, the product is generally calibrated. Do not use despeckling before, otherwise, it would be lossy preprocessing. You can incorporate the corrections of terrain, and noise in post-processing.

In this library, you have to create 4 * 2 channels (Amplitude and Phase as separate) for the full Polarimetric dataset as HH, HV, VH, and VV.

**Comment by chetan95** [comment]
Hi @raktim-ghosh , thanks for the response. I am using raw imagery, calibrating it using SNAP then I  have created a stack of 8 images as you mentioned. I believe the end result is not multi-looked, am I understanding it correct?

**Comment by raktim-ghosh** [comment]
Hi @chetan95, yes the end product is basically the Single Look Complex (SLC). The end result is not multi-looked. However, you can modify the library after generating the SLC, by let's say incorporating different tilting of the Synthetic Apertures with rotation matrices. Generally, these tilting can help mitigate the ambiguity of scattering mechanisms. I didn't work on multi-looking for this library. Let me know if you need anything further.

**Comment by raktim-ghosh** [comment]
Hi @chetan95 just ignore my previous comment. Actually, I was not in touch with the literature for a long time. If you take a spatial averaging with (5 * 5 or 7 * 7), this is a process of multi-looking. In my library, I used Gaussian Blur, meaning I took a spatial averaging with a square window (w * w). Actually, this is a multilooking. Read this paper. It can clarify your doubt. https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8894523. If you don't take the spatial averaging, the end result is a SLC.
